packages/hardhat-zksync-chai-matchers/README.md

```diff
@@ -0,0 +1,3 @@
+# hardhat-zksync-chai-matchers
+
+[Hardhat](https://hardhat.org/) plugin that adds zkSync-specific capabilities to the [Chai](https://chaijs.com/) assertion library, making your smart contract tests easy to write and read. 
```

Comment:
I would like to see the documentation for the provided matchers.
Either here or in our public docs (then the link to the public documentation should be present here).

Reply:
We definitely need to work on the documentation for all the updated plugins :)